src/site/header.rs

```diff
@@ -41,7 +41,7 @@ pub fn create(config: &Config) -> Result<Box<header<String>>> {
                 let mut href = format!("/{}", file_name);
 
                 if let Some(prefix) = &config.path_prefix {
-                    href = format!("/{}{}", prefix, href);
```

Comment:
This seems to be a fix but it's super cumbersome

https://github.com/tokio-rs/axum/discussions/446

Reply:
oh interesting, def a bummer. might be a nice opportunity to contribute back. regardless maybe we can also snag some of the error work from that answer so we don't just serve the empty page if there's an issue?

Reply:
We still need this right?

Reply:
no i dont think so, i ran it locally and it worked?